src/transformers/models/poolformer/modeling_poolformer.py

```diff
@@ -44,11 +44,11 @@
 
 # Base docstring
 _CHECKPOINT_FOR_DOC = "sail/poolformer_s12"
-_EXPECTED_OUTPUT_SHAPE = [1, 197, 768]
+_EXPECTED_OUTPUT_SHAPE = [1, 512, 7, 7]
```

Comment:
@NielsRogge - this was failing as well. Is the new output shape ok?

Reply:
Yes that looks ok, thanks for fixing.